### PR TITLE
fixing broken links

### PR DESCRIPTION
**Pull request by chrisdmills-okta**
## Description:
- **What's changed?** Fixing some broken links that have appeared since some of the multi-page to single page work.
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-443780](https://oktainc.atlassian.net/browse/OKTA-443780)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/unlock-mobile-app-with-biometrics/main/index.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ In this guide you will learn how to integrate biometric authentication like Face
 
 **What you need**
 
-* Biometric unlock with Touch ID, Face ID, and Smart Lock — added by following the [Sign users in to your mobile apps](/docs/guides/sign-into-mobile-app/android/before-you-begin/) guide.
+* Biometric unlock with Touch ID, Face ID, and Smart Lock — added by following the [Sign users in to your mobile apps](/docs/guides/sign-into-mobile-app/android/main/) guide.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/unlock-mobile-app-with-biometrics/main/index.md:19]:
```suggestion
* Biometric unlock with Touch ID, Face ID, and Smart Lock — added by following the [Sign users in to your mobile apps](/docs/guides/sign-into-mobile-app/) guide.
```

File: packages/@okta/vuepress-site/docs/guides/unlock-mobile-app-with-biometrics/main/index.md [hunk 2]
```diff
@@ -27,7 +27,7 @@ In this guide you will learn how to integrate biometric authentication like Face
 
 ## Overview
 
-When you [configure the Okta SDK](/docs/guides/sign-into-mobile-app/configure-packages/) with the `offline_access` scope, your mobile app gets a refresh token from Okta. The refresh token is long-lived and is used to keep the user signed in to your app.
+When you [configure the Okta SDK](/docs/guides/sign-into-mobile-app/android/main/#add-and-configure-packages) with the `offline_access` scope, your mobile app gets a refresh token from Okta. The refresh token is long-lived and is used to keep the user signed in to your app.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/unlock-mobile-app-with-biometrics/main/index.md:30]:
```suggestion
When you [configure the Okta SDK](/docs/guides/sign-into-mobile-app/-/main/#add-and-configure-packages) with the `offline_access` scope, your mobile app gets a refresh token from Okta. The refresh token is long-lived and is used to keep the user signed in to your app.
```
Both this page and the /sign-into-mobile-app/ page contain the same stack selectors, so we can't hardcode "android".

File: packages/@okta/vuepress-site/docs/guides/build-custom-ui-mobile/create-an-okta-application/index.md [hunk 3]
```diff
@@ -3,6 +3,6 @@ title: Create an Okta application
 ---
 Before you can sign a user in, you need to create an Okta application that represents your mobile application.
 
-Use the Okta application that you created when you walked through the [Sign users in to your mobile app](/docs/guides/sign-into-mobile-app/create-okta-application/) guide.
+Use the Okta application that you created when you walked through the [Sign users in to your mobile app](/docs/guides/sign-into-mobile-app/main/) guide.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/build-custom-ui-mobile/create-an-okta-application/index.md:6]:
```suggestion
Use the Okta application that you created when you walked through the [Sign users in to your mobile app](/docs/guides/sign-into-mobile-app/) guide.
```
Removing "/main/" is cleaner. Plus, it avoids the broken nav and bread-crumbs bug.

File: packages/@okta/vuepress-site/docs/guides/build-custom-ui-mobile/configure-packages/index.md [hunk 4]
```diff
@@ -3,7 +3,7 @@ title: Add and configure packages
 ---
 Next, you need to add Okta to your application by installing and configuring our SDK.
 
-You should already have added and configured packages when you walked through the [Sign users in to your mobile app](/docs/guides/sign-into-mobile-app/android/configure-packages/) guide.
+You should already have added and configured packages when you walked through the [Sign users in to your mobile app](/docs/guides/sign-into-mobile-app/android/main/) guide.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/build-custom-ui-mobile/configure-packages/index.md:6]:
```suggestion
You should already have added and configured packages when you walked through the [Sign users in to your mobile app](/docs/guides/sign-into-mobile-app/-/main/#add-and-configure-packages) guide.
```

This breaks the left-nav and breadcrumbs, but at least it's jumping to the right section with the selected language preserved.

**Review by vanngo-okta — CHANGES_REQUESTED** [review]
A few nits.

**Comment by chrisdmills-okta** [comment]
@vanngo-okta suggestions all committed. Thanks!

**Review by vanngo-okta — APPROVED** [review]
LGTM! 🚀

**Pull request merged**